gammapy/irf/psf_table.py

```diff
@@ -567,12 +568,16 @@ def containment_radius(self, energies, fraction, interp_kwargs=None):
         rad : `~astropy.units.Quantity`
             Containment radius in deg
         """
-        # TODO: figure out if there's a more efficient implementation to support
-        # arrays of energy
-        energies = np.atleast_1d(energies)
-        psfs = [self.table_psf_at_energy(energy, interp_kwargs) for energy in energies]
-        rad = [psf.containment_radius(fraction) for psf in psfs]
-        return Quantity(rad)
+        # oversample for better precision
```

Comment:
Shouldn't the `containment_radius` of the `TablePSF` be adapted too, to follow the same approach?
This would completely remove the need for spline parametrization.

Reply:
I agree this is a good suggestion, but I'd prefer to do this in a separate PR.